pkg/agent/datamodel/sig_config.go

```diff
@@ -432,6 +437,22 @@ var (
 		Version:       EdgeZoneSIGImageVersion,
 	}
 
+	// This image is using a specific resource group and gallery name for edge zone scenario.
+	SIGUbuntuEdgeZoneContainerd2204ImageConfigTemplate = SigImageConfigTemplate{
+		ResourceGroup: AKSUbuntuEdgeZoneResourceGroup,
+		Gallery:       AKSUbuntuEdgeZoneGalleryName,
+		Definition:    "2204containerd",
+		Version:       EdgeZoneSIGImageVersion,
+	}
+
+	// This image is using a specific resource group and gallery name for edge zone scenario.
+	SIGUbuntuEdgeZoneContainerd2204Gen2ImageConfigTemplate = SigImageConfigTemplate{
```

Comment:
suggest making these private as they arent exported, or else can add //nolint if they must be exported (I don't believe so?)

Reply:
fixing existing code in https://github.com/Azure/AgentBaker/pull/2976/files#diff-7666a5e71fa9f1c54fbb8e13ce2b0b970f339aff9444248e0513467f834a8052